Maquetado/views/login.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>Login</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="../application.css">
+</head>
+
+<body>
+  <div class='pageContainer'>
+    <div class='loginContainer'>
+      <img src='../resources/img/LogoWolox.png' class='woloxLoginLogo' alt='Wolox Logo Image' />
+      <div class='inputsContainer'>
+        <h2 class='dataTitle'>Email</h2>
+        <input class='dataInput'>
+        <h2 class='dataTitle'>Password</h2>
+        <input class='dataInput'>
+        <div class='loginButton'>
+          <a href='http://127.0.0.1:5500/Maquetado/views/bookList.html'>
+            <h3 class='loginTitle'>Login</h3>
```

Comment:
Using a form is usual to use a button to post the data. like this.
```suggestion
          <button type='submit'>
            <h3 class='loginTitle'>Login</h3>
```